.github/workflows/contracts.yml

```diff
@@ -67,4 +63,4 @@ jobs:
           restore-keys: ${{ runner.os }}-cargo-build
 
       - name: Test
-        run: cargo test --verbose
+        run: cargo test
```

Comment:
Ok, just added the verbose flag because in previous projects when something goes wrong, maybe we dont have enough info. But i think we can give it a try in this way and change later.